script.js

```diff
@@ -23,4 +21,22 @@ tags.forEach((tag) => {
   tag.addEventListener('click', () => {
     closeMenu();
   });
-});
\ No newline at end of file
+});
+
+const form = document.querySelector('#form');
+
+form.addEventListener('submit', (e) => {
+  // Prevent form submission
+  e.preventDefault();
+
+  // Validate form fields
+  const email = document.querySelector('#user-email').value;
+
+  // Check if name is empty
+  if (email !== email.toLowerCase()) {
+    const errorDiv = document.querySelector('#email-error');
+    errorDiv.innerText = 'The email address must be lowercase';
+  } else {
+    form.submit();
+  }
+});
```

Comment:
- [ ] [OPTIONAL] **_This is only marked as optional because the previous reviewer did not mention this issue._** Your form validations are not working as expected. Kindly fix this as it is the main requirement for this project. The previous reviewer may have not noticed that the validation is not working and that users can submit the form even when the email address has a CAPITAL letter. 

Below is how your contact section is after your project loads. Notice the **_red_** text .
![image](https://github.com/Paulinelali/microverse-personal-portfolio/assets/84156714/b09ef67a-6408-4f5f-ae4d-40f0ffe4457f)

Reply:
hello @bobb-Rob 
Thank you for taking the time to review my project.
However, you are viewing the wrong branch or you are probably doing something I don't understand.
Please see the attached scroonshot which is my current form after the previous reviewer asked for a correction.
Please check and verify so I can do the necessary corrections or you can give it APPROVED
Thank you😎[Capture](https://github.com/Paulinelali/microverse-personal-portfolio/assets/88007635/3e98842b-ab80-4223-a8cc-fb0c0a5dcb09)

Reply:
@Shabiblac, It works now. Well done.

You can ask for a review on your dashboard and your project will be approved.